client/src/pages/TotalExpensesChart.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Typography, Divider } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import Paper from "@material-ui/core/Paper";
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  SplineSeries
+} from "@devexpress/dx-react-chart-material-ui";
+
+const useStyles = theme => ({
+  root: {
+      flexGrow: 1
+  }
+});
+
+class TotalExpensesChart extends Component {
+  state = {
+      daily_expenses: [{expense: 10.01, date: "Nov 3"},
```

Comment:
do you need this mock data now that you're fetching?

Reply:
I'll remove it later!